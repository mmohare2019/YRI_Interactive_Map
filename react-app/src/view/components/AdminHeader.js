import React from "react";
import Container from 'react-bootstrap/Container';
import Navbar from 'react-bootstrap/Navbar';

function AdminHeader() {
  return (
    <>
      <Navbar className="header" variant="light">
        <Container>
          <Navbar.Brand href="/admin">
            <img className="yri-logo" src="YRI_LOGO.png" width="30" height="30" alt="YRI logo"/>
            <img className="ccsj-logo" src="ccsjLogo.jpg" width="30" height="30" alt="CCSJ logo"/>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
                <span class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
                <div class="navbar-nav">
                <a class="nav-item nav-link" href="/admin">Home</a>
                <a class="nav-item nav-link" href="/inbox">Messages</a>
                <a class="nav-item nav-link" href="/partners">Partners</a>
                <a class="nav-item nav-link" href="/category">Categories</a>
                </div>
            </div>
          </Navbar.Brand>
        </Container>
      </Navbar>
      <br/>
    </>
  );
}

export default AdminHeader;