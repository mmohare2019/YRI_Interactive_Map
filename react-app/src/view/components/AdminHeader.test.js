import TestRenderer from "react-test-renderer";
import AdminHeader from "./AdminHeader";
import React from "react";
import renderer from 'react-test-renderer';

describe("admin header tests", () => {
    test("render smoke test", () => {
        TestRenderer.create(<AdminHeader/>)
    })
});

test("admin header test", function () {
    const component = renderer.create(<AdminHeader/>).toJSON();
    expect(component).toMatchSnapshot(); 
});