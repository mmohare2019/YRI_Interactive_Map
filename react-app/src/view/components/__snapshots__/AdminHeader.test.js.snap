// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`admin header test 1`] = `
Array [
  <nav
    className="header navbar navbar-expand navbar-light"
  >
    <div
      className="container"
    >
      <a
        className="navbar-brand"
        href="/admin"
      >
        <img
          alt="YRI logo"
          className="yri-logo"
          height="30"
          src="YRI_LOGO.png"
          width="30"
        />
        <img
          alt="CCSJ logo"
          className="ccsj-logo"
          height="30"
          src="ccsjLogo.jpg"
          width="30"
        />
        <button
          aria-controls="navbarNavAltMarkup"
          aria-expanded="false"
          aria-label="Toggle navigation"
          class="navbar-toggler"
          data-target="#navbarNavAltMarkup"
          data-toggle="collapse"
          type="button"
        >
          <span
            class="navbar-toggler-icon"
          />
        </button>
        <div
          class="collapse navbar-collapse"
          id="navbarNavAltMarkup"
        >
          <div
            class="navbar-nav"
          >
            <a
              class="nav-item nav-link"
              href="/admin"
            >
              Home
            </a>
            <a
              class="nav-item nav-link"
              href="/inbox"
            >
              Messages
            </a>
            <a
              class="nav-item nav-link"
              href="/partners"
            >
              Partners
            </a>
            <a
              class="nav-item nav-link"
              href="/category"
            >
              Categories
            </a>
          </div>
        </div>
      </a>
    </div>
  </nav>,
  <br />,
]
`;
